content/integrations/activemq.md

```diff
@@ -9,103 +9,23 @@ description: "{{< get-desc-from-git >}}"
 ---
 
 ## Overview
-
-Get metrics from ActiveMQ in real time to
-
-* Visualize your web ActiveMQ server performance
-* Correlate the performance of ActiveMQ with the rest of your applications
+{{< get-overview-from-git >}}
```

Comment:
@l0k0ms - this is all looking good. We should decide on a different pattern for front-matter placeholders. The pattern you are using now {{< get-desc-from-git >}} is currently being used by Hugo to designate shortcodes, and so may be confusing to other developers. In the python script you are using this pattern for substitution. Maybe something like << desc-from-git >> ?

Reply:
👍